db/migrations/01_users_table_setup.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE users(id SERIAL PRIMARY KEY, email varchar(60), username varchar(60), password varchar(60));
```

Comment:
Could include First name and Last name fields on sign up to meet the specification.